app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -54,7 +54,12 @@ div.panel.panel-default
           td = format_datetime(@submission.created_at)
         tr
           th = @submission.class.human_attribute_name(:submitted_at)
-          td = format_datetime(@submission.submitted_at)
+          - if @assessment.end_at && @submission.submitted_at > @assessment.end_at
+            td.bg-danger.text-danger
+              span.text-danger.glyphicon.glyphicon-exclamation-sign title=t('.late_submission_warning')
```

Comment:
Small issue, but can I suggest to shift the tooltip to behind the date (so that I can compare dates from row to row easily. 

I find the red background jarring though, how about bolded + red text instead?